### ISSUE TITLE
Jolt\Core\Reference.h(116): warning C5266: 'const' qualifier on return type has no effect

### ISSUE THREAD

**Issue by tksuoran** [jrouwe/JoltPhysics#549]
```C++
/// Casting operators
inline					operator T * const () const						{ return mPtr; }
inline					operator T *()									{ return mPtr; }
```
I wonder if the `const` should be moved a bit to left or something.

**Comment by mihe** [comment]
This was fixed in #533 already.

**Comment by tksuoran** [comment]
Sorry, this was my bad, I needed to rerun configure script on my project to get latest versions of dependencies. Thanks for the prompt response!
